bin/help/getRenderProperties.js

```diff
@@ -49,7 +49,7 @@ function getRenderProperties(element) {
 				return newCanvasRenderProperties(element);
 			}
 			// If SVG
-			else if (element && element.nodeName === 'svg' || typeof SVGElement !== 'undefined' && element instanceof SVGElement) {
```

Comment:
If `element` exist but `nodeName` does not, this will crash

Reply:
right you are. I'll fix.